### DISCUSSION TITLE
Recycle Bin Option

### DISCUSSION THREAD

**Discussion by connectshikha** [manugarg/pagenotes-issues#47] [General]
Hi,
Can you please add a recycle bin option for the deleted notes?
Sometimes, we may delete a note and later need to relook/check it.

Also, I have observed that when I am making notes on different video(s) over YouTube, all the notes are combined and shown in one single note with the website www.youtube.com instead of separate URL addresses of those respective videos(s).

Thanks

**Comment by manugarg** [comment]
@connectshikha I'll consider adding the recycle-bin option. I'll have to think more about the design.

Regarding notes association, you should be able to select whether you want the note to be attached to the whole site or a specific page.

You can either do it per note:
<img width="331" alt="image" src="https://github.com/manugarg/pagenotes-issues/assets/1286425/bb53dce8-6758-4b97-a017-f7a0b01bf499">

Or, you can change default in the extension options:
<img width="1134" alt="image" src="https://github.com/manugarg/pagenotes-issues/assets/1286425/482da215-a322-4cfd-9279-42c58c5fe966">

Hope that helps!

**Comment by manugarg** [comment]
@connectshikha I wanted to update you regarding the youtube video notes issue. Now Page Notes handles youtube videos notes as a special case and it should always work. Please give it a try with version >= 3.4.14.

Related: https://github.com/manugarg/pagenotes-issues/issues/101

While I might still have your attention:
I've been working on improving the [Page Notes app](https://pagenotes.app) as well, to give users access to their notes from any device. If it fits your use case, please give  ([pagenotes.app](https://pagenotes.app)) a try on your phone and let me know if you have any feedback.
